docs/stratis.txt

```diff
@@ -42,11 +42,20 @@ pool destroy <pool_name>::
 pool add-data <pool_name> <blockdev> [<blockdev>..]::
 	 Add one or more blockdevs to an existing pool, to enlarge its storage
 	 capacity.
+pool init-cache <pool_name> <blockdev> [<blockdev>..]::
+	 Initialize a cache for an existing pool.
+	 Add one or more blockdev to a pool, to be used as cache
+	 instead of additional storage. Typically, smaller and faster
+	 drives such as SSDs, are used for this purpose.
+	 NOTE: There is a temporary restriction on the total size of the
```

Comment:
Please create a new sub-heading for this note, called something like ```RESTRICTIONS``` and put this cache size restriction there. You could probably put it above the ```NOTES``` heading, which is purely informational, and just common sense, really. Then just omit the note from these two cache-related commands.